packages/cli/src/environments.ee/source-control/source-control-import.service.ee.ts

```diff
@@ -94,6 +225,22 @@ export class SourceControlImportService {
 				if (!remote?.id) {
 					return undefined;
 				}
+
+				if (workflowFilter) {
+					// If we need to apply the filter
+					if (
+						!workflowFilter.some((project) => {
+							if (remote.owner.type === 'team') {
+								return project.id === remote.owner.teamId;
```

Comment:
I'm just thinking, with this filter, it means that workflow that changed projects in the remote will not be returned.
Thus, it will be in the `wfMissingInRemote` variables, the modal will display the workflow as new to be pushed, and the workflow might be pushed to the remote and be "duplicated", no ?

Reply:
It would be shown as new yes, but it wouldn't duplicate it, because the workflow files are stored by id name. So for the user it would look like a new workflow yes, but actually it would only be an update. 👍

Reply:
Ah yes, thank you for clarifying :+1: